test/integration/backward_compatible/parallel/flakeid/FlakeIdGeneratorOutOfRangeTest.js

```diff
@@ -19,18 +19,24 @@ const chai = require('chai');
 chai.use(require('chai-as-promised'));
 const expect = chai.expect;
 
-const RC = require('../../RC');
-const { Client, HazelcastError } = require('../../../../');
-const TestUtil = require('../../../TestUtil');
+const RC = require('../../../RC');
+const { HazelcastError } = require('../../../../../lib');
+const TestUtil = require('../../../../TestUtil');
+const { assertTrueEventually } = require('../../../../TestUtil');
 
 describe('FlakeIdGeneratorOutOfRangeTest', function () {
     let cluster, client;
     let flakeIdGenerator;
+    const testFactory = new TestUtil.TestFactory();
 
     afterEach(async function () {
         await flakeIdGenerator.destroy();
         await client.shutdown();
-        return RC.terminateCluster(cluster.id);
+        await RC.terminateCluster(cluster.id);
```

Comment:
Yeah I can do that. But to be sure there is no leak, I would like to keep the `after` hook because basically there is one in every `describe` block. If there are no clients or clusters, it is no-op.

Reply:
I feel like the `after` block in this test is not needed as there is a `shutdownAll` in `afterEach`

Reply:
Like I said, all describe blocks have a after with shutdownAll and I want to keep it to be sure there is no leak. I don't know maybe `await flakeIdGenerator.destroy();` will throw and we will leak a client or a cluster.

Reply:
ok deleted `after`